BATTLES.md

```diff
@@ -0,0 +1,66 @@
+# Battles
+
+A guide to battles.
```

Comment:
Follow precedent from other docs files and be more descriptive about what this guide will contain.